packages/airnode-abi/src/constants.ts

```diff
@@ -0,0 +1,21 @@
+import { ValueOf, TypeTransformation } from './types';
+
+// Upper case letters refer to dynamically sized types, lower case letters refer to statically sized types
+export const PARAMETER_SHORT_TYPES = {
+  B: 'bytes',
+  b: 'bytes32',
+  S: 'string',
+  s: 'string32', // This is not officially supported type in solidity and we use bytes32 on chain instead
```

Comment:
Yeah, I've explained this more with this comment:
```
  // Type "string32" is not supported in solidity and we use bytes32 on chain instead. We need to create this
  // artificially made up type so that users can decide if they want their data encoded as strings or as raw bytes
  // value.
```